rs/nervous_system/tools/sync-with-released-nervous-system-wasms/src/main.rs

```diff
@@ -41,15 +44,29 @@ async fn get_mainnet_canister_git_commit_id_and_module_hash(
         git_commit_id.pop();
     }
 
-    let module_hash = agent
-        .read_state_canister_info(canister_id, "module_hash")
-        .await?;
-    let module_hash = module_hash.iter().fold(String::new(), |mut output, x| {
+    let module_hash = if canister_name.ends_with("_test") {
+        // we get the module hash of a test canister from AWS
+        let module_url = format!(
+            "https://download.dfinity.systems/ic/{git_commit_id}/canisters/{canister_name}.wasm.gz"
```

Comment:
This relies on the assumption that the canister name in the json file is the same as the canister name pushed to AWS. This assumption reduces complexity (otherwise, we'd have to load the [mapping](https://github.com/dfinity/ic/blob/d3665366e20bf83667bc3718d7ad988876966034/WORKSPACE.bazel#L19-L52) from the bazel workspace), but I wonder if it should be documented in some form.